State/Merchant.ts

```diff
@@ -35,7 +36,8 @@ export namespace Merchant {
 			typeof value.refundable == "number" &&
 			typeof value.captured == "number" &&
 			typeof value.settled == "number" &&
-			typeof value.fees == "number"
+			typeof value.fees == "number" &&
+			(value.authorized == undefined || typeof value.fees == "number")
```

Comment:
Also create new PR on master and fix this on master:
```suggestion
			(value.authorized == undefined || typeof value.authorized == "number")
```

Reply:
Yes, I fixed it, needed it fixed